src/components/AddItemForm.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { withFirestore } from 'react-firestore';
+import '../CSS/AddItemForm.css';
```

Comment:
Now that #5 is merged, you could add `useToken` to this component and utilize the same token that the user has created and saved to `localStorage`.